orttraining/orttraining/test/python/_test_helpers.py

```diff
@@ -231,6 +231,22 @@ def assert_values_are_close(input, other, rtol=1e-05, atol=1e-06):
         assert False, err_msg
 
 
+def check_pythonop_training_mode(model, is_eval_mode):
```

Comment:
I think you can add the two PythonOp related tests in orttraining/orttraining/test/python/orttraining_test_ortmodule_autograd.py. One is
test_training_mode_export(), the other one is test_eval_mode_export(). Then you don't need modify orttraining/orttraining/test/python/_test_helpers.py (which is a test helper).

Reply:
i add a test "test_pythonop_training_mode" to check both training and eval mode